modules/ROOT/pages/salesforce/salesforce-mktg-connector-reference.adoc

```diff
@@ -1,661 +1,778 @@
+| Key Password a| String | The password used to protect the private key. |  |
+| Password a| String | The password used to protect the key store. |  |
+| Algorithm a| String | The algorithm used by the key store. |  |
+|===
 
-[[new-object-trigger]]
-=== On New Object
-`<sfdc-marketing-cloud:new-object-trigger>`
-
+[[StandardRevocationCheck]]
+=== Standard Revocation Check
 
-==== Parameters
-[%header,cols="20s,20a,35a,20a,5a"]
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
-| Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
-| Since a| String |  +++Specify a date in the yyyy-MM-dd HH:mm:ss (that is, 2017-03-17 16:30:40) to retrieve the selected objects+++ |  |
-| Object Type a| String |  |  | x
-| Primary Node Only a| Boolean |  +++Whether this source should only be executed on the primary node when running in Cluster.+++ |  |
-| Scheduling Strategy a| scheduling-strategy |  +++Configures the scheduler that triggers the polling.+++ |  | x
-| Redelivery Policy a| <<RedeliveryPolicy>> |  +++Defines a policy for processing the redelivery of the same message.+++ |  |
-| Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  |
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean | Only verify the last element of the certificate chain. |  |
+| Prefer Crls a| Boolean | Try CRL instead of OCSP first. |  |
+| No Fallback a| Boolean | Do not use the secondary checking method (the one not selected before). |  |
+| Soft Fail a| Boolean | Avoid verification failure when the revocation server cannot be reached or is busy. |  |
 |===
 
-==== Output
-[%autowidth.spread]
+[[CustomOcspResponder]]
+=== Custom Ocsp Responder
+
+[%header,cols="20s,25a,30a,15a,10a"]
 |===
-|Type |Object
-| Attributes Type a| Any
+| Field | Type | Description | Default Value | Required
+| Url a| String | The URL of the OCSP responder. |  |
+| Cert Alias a| String | Alias of the signing certificate for the OCSP response (must be in the trust store), if present. |  |
 |===
 
-==== For Configurations
-* <<sfdc-marketing-cloud-config>>
-
+[[CrlFile]]
+=== Crl File
 
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The path to the CRL file. |  |
+|===
 
-== Types
 [[Reconnection]]
 === Reconnection
 
 [%header,cols="20s,25a,30a,15a,10a"]
 |===
 | Field | Type | Description | Default Value | Required
 | Fails Deployment a| Boolean | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> | The reconnection strategy to use. |  |
+| Reconnection Strategy a| * <<Reconnect>>
```

Comment:
Update the Reconnect and Reconnect Forever blocks per Step 8 of our reference instructions.